drivers/hmis/app/graphql/schema.graphql

```diff
@@ -3055,6 +3070,11 @@ type Query {
   """
   projectCoc(id: ID!): ProjectCoc
 
+  """
+  Search for projects
+  """
+  projectSearch(input: ProjectSearchInput!, limit: Int, offset: Int, projectTypes: [ProjectType!], sortOrder: ProjectSortOption): ProjectsPaginated!
```

Comment:
I guess `projectTypes` is showing up here because it's on `has_projects`? For another day, but we can remove that (it's unused right now) and move it into `ProjectSearchInput`. We can also add Organization ID into `ProjectSearchInput`, and thats all we should need for https://www.pivotaltracker.com/story/show/184091576